### ISSUE TITLE
"OpenStreetMap file you just downloaded is nothing but an inscrutable hunk of bits" is going a bit too far

### ISSUE THREAD

**Issue by matkoniecz** [clarisma/geodesk-site#5]
https://docs.geodesk.com/why-geodesk claims

> The crux of the problem is this: That shiny OpenStreetMap file you just downloaded is nothing but an inscrutable hunk of bits until you load it into a database, where you can dissect it with spatial queries.

that is not really true. Especially smaller files can be read by humans or naive scripts and processed without involving databases at all.

**Comment by clarisma** [comment]
Agree, the tone is a bit hyperbolic. What we had in mind were PBF files, country-sized or bigger, which aren't human-readable and are too large to be processed in-memory on most systems. The text would be stronger with specific examples.

I've created a new issue (#7) to review the documentation for style and clarity in general. There will be some significant updates as we move to Version 0.2, so the docs are slated for an overhaul in any case.

Thanks for bringing this to our attention!
